src/Stratis.CirrusPegD/Program.cs

```diff
@@ -79,8 +99,12 @@ private static IFullNode GetSidechainFullNode(string[] args)
                 MinProtocolVersion = ProtocolVersion.ALT_PROTOCOL_VERSION
             };
 
+            var fedPegOptions = new FederatedPegOptions(
```

Comment:
Can this include `federationPubKeys`?

Reply:
Ah yeah it could eventually. At the moment the functionality is split between FederatedPegOptions and FederationGatewaySettings, so we will have to work out the long-term strategy there

Reply:
Long term we may be looking at a '-federationPubKey` command line option to feed into the above list via a `GetAll`. What we have currently constitutes application-level defaults.